tool/src/main.rs

```diff
@@ -601,6 +602,13 @@ fn main() {
                 process::exit(1);
             },
         },
+        Some(("reset", _sub_m)) => match unsafe { ec.reset() } {
+            Ok(()) => (),
+            Err(err) => {
+                eprintln!("failed to send reset command: {:X?}", err);
```

Comment:
Is it only *sending* the command that will fail? What about: "failed to reset device"